src/libcore/default.rs

```diff
@@ -115,6 +115,48 @@ pub trait Default: Sized {
     fn default() -> Self;
 }
 
+/// Equivalent to [`Default::default()`] but without repeating the word
+/// "default" twice at the call site.
+///
+/// For example:
+/// ```
+/// #![feature(default_free_fn)]
+///
+/// use std::default::default;
+///
+/// #[derive(Default)]
+/// struct AppConfig {
+///     foo: FooConfig,
+///     bar: BarConfig,
+/// }
+///
+/// #[derive(Default)]
+/// struct FooConfig {
+///     foo: i32,
+/// }
+///
+/// #[derive(Default)]
+/// struct BarConfig {
+///     bar: f32,
+///     baz: u8,
+/// }
+///
+/// fn main() {
+///     let options = AppConfig {
+///         foo: default(),
+///         bar: BarConfig {
+///             bar: 10,
+///             ..default(),
+///         },
+///     };
+/// }
+/// ```
+#[unstable(feature = "default_free_fn", issue = "none")]
```

Comment:
Please open a tracking issue (https://github.com/rust-lang/rust/issues/new/choose) and add the issue number here in place of `none`.

Reply:
There is no RFC for this change.  Do I need to create one or is it OK not to have one?

Reply:
The instructions say that I need to apply labels to the tracking issue, but I do not see a UI to do so.  Is it that I do not have permissions?

Tracking issue is here: https://github.com/rust-lang/rust/issues/73014

Reply:
I think https://github.com/rust-lang/triagebot/wiki/Labeling is the way anyone can apply labels without particular permissions.

Reply:
Thanks for the pointer.
According to the docs, I need to mark the request with a new label, matching the new feature name: F-default_free_fn.
I can not create a new label - the "New Label" button is not visible in the UI.
The bot, I guess, can only add a label that already exists.